modules/common/istio/istio-values.tpl

```diff
@@ -194,12 +203,3 @@ sidecarInjectorWebhook:
     limits:
       cpu: 25m
       memory: 64Mi
-
-security:
```

Comment:
why remove the requests / limits for citadel?

Reply:
I saw a duplicate of the security limits / requests so I removed one of them

Reply:
ah, I didn't catch that.  did you want to modify the requests / limits for the one you kept?